### PR TITLE
Updated per BZ1801331

### PR DESCRIPTION
**Pull request by sagidlow**
Applies to versions 4.4+
BZ Link: https://bugzilla.redhat.com/show_bug.cgi?id=1801331
QE Review: @weinliu

### CODE REVIEW

File: applications/projects/working-with-projects.adoc [hunk 1]
```diff
@@ -13,6 +13,11 @@ isolation from other communities.
 Projects starting with `openshift-` and `kube-` are  xref:../../authentication/using-rbac.adoc#rbac-default-projects_using-rbac[default projects]. These projects host cluster components that run as pods and other infrastructure components. As such, {product-title} does not allow you to create projects starting with `openshift-` or `kube-` using the `oc new-project` command. Cluster administrators can create these projects using the `oc adm new-project` command.
 ====
 
+[NOTE]
+====
+You cannot assign a SCC to pods created in one of the default namespaces: `default`, `kube-system`, `kube-public`, `openshift-node`, `openshift-infra`, `openshift`. These namespaces should not be used for running pods or services.
```
Comment (kalexand-rh) [inline applications/projects/working-with-projects.adoc:18]:
```suggestion
You cannot assign an SCC to pods created in one of the default namespaces: `default`, `kube-system`, `kube-public`, `openshift-node`, `openshift-infra`, and `openshift`. You cannot use these namespaces for running pods or services.
```

Comment (sagidlow) [reply applications/projects/working-with-projects.adoc]: Added the changes to each changed file.

**Pull request merged**